stdlib/public/Reflection/TypeRefBuilder.cpp

```diff
@@ -380,89 +378,89 @@ void TypeRefBuilder::dumpAssociatedTypeSection(std::ostream &OS) {
                          readTypeRef(descriptor, descriptor->ProtocolTypeName));
       auto protocolName = nodeToString(protocolNode);
 
-      OS << "- " << conformingTypeName << " : " << protocolName;
-      OS << '\n';
+      fprintf(file, "- %s : %s", conformingTypeName.c_str(), protocolName.c_str());
+      fprintf(file, "\n");
 
       for (const auto &associatedTypeRef : *descriptor.getLocalBuffer()) {
         auto associatedType = descriptor.getField(associatedTypeRef);
         
         std::string name = getTypeRefString(
                             readTypeRef(associatedType, associatedType->Name));
-        OS << "typealias " << name << " = ";
+        fprintf(file, "typealias %s = ", name.c_str());
         dumpTypeRef(
-          readTypeRef(associatedType, associatedType->SubstitutedTypeName), OS);
+          readTypeRef(associatedType, associatedType->SubstitutedTypeName), file);
       }
     }
   }
 }
 
-void TypeRefBuilder::dumpBuiltinTypeSection(std::ostream &OS) {
+void TypeRefBuilder::dumpBuiltinTypeSection(FILE *file) {
   for (const auto &sections : ReflectionInfos) {
     for (auto descriptor : sections.Builtin) {
       auto typeNode = demangleTypeRef(readTypeRef(descriptor,
                                                   descriptor->TypeName));
       auto typeName = nodeToString(typeNode);
       
-      OS << "\n- " << typeName << ":\n";
-      OS << "Size: " << descriptor->Size << "\n";
-      OS << "Alignment: " << descriptor->getAlignment() << "\n";
-      OS << "Stride: " << descriptor->Stride << "\n";
-      OS << "NumExtraInhabitants: " << descriptor->NumExtraInhabitants << "\n";
-      OS << "BitwiseTakable: " << descriptor->isBitwiseTakable() << "\n";
+      fprintf(file, "\n- %s:\n", typeName.c_str());
+      fprintf(file, "Size: %s\n", std::to_string(descriptor->Size).c_str());
+      fprintf(file, "Alignment: %s:\n", std::to_string(descriptor->getAlignment()).c_str());
+      fprintf(file, "Stride: %s:\n", std::to_string(descriptor->Stride).c_str());
+      fprintf(file, "NumExtraInhabitants: %s:\n", std::to_string(descriptor->NumExtraInhabitants).c_str());
```

Comment:
These are all numeric values, why go through `%s`?

Reply:
Oh, my bad 🧐
I'll fix it

Reply:
Done